src/modules/localization/domain/dynamics/constant_acceleration_model.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef LOCALIZATION__DYNAMICS__DYNAMICS_HPP
+#define LOCALIZATION__DYNAMICS__DYNAMICS_HPP
+
+#include<eigen3/Eigen/Dense>
+
+namespace localization::dynamics {
+    /**
+     * @brief A constant acceleleration model based on particle physics
+     * 
+     */
+    class ConstantAccelerationModel{
+        public:
+        /**
+         * @brief Construct a new Constant Acceleration Model object
+         * 
+         * @param[in] state_covariance_x 
+         * @param[in] state_covariance_y 
+         * @param[in] state_covariance_theta 
```

Comment:
Parameters not being taken as input can be removed

Reply:
good catch